### PR TITLE
Resolve IDENT-3075: Added remove profile methoed to askar profile

### PR DESCRIPTION
**Pull request by dejsenlitro**

### CODE REVIEW

File: aries_cloudagent/askar/tests/test_profile.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        openStore = AskarOpenStore
+        context = InjectionContext() 
+        context.settings = {
+            "multitenant.wallet_type": "askar-profile"
+        }
+        askar_profile = AskarProfile(
+            openStore,
+            context
+        )
+
```
Comment (vidp1) [inline aries_cloudagent/askar/tests/test_profile.py:28]: pls delete line if posssible.

File: aries_cloudagent/askar/tests/test_profile.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        remove_profile_stub = asyncio.Future()
+        remove_profile_stub.set_result(True)
+        openStore.store.remove_profile.return_value = remove_profile_stub
+
+        await askar_profile.remove()
+        
+        openStore.store.remove_profile.assert_called_once()
+
+    @mock.patch("aries_cloudagent.askar.store.AskarOpenStore")
+    async def test_remove_fail_if_wallet_type_not_askar_profile(self, AskarOpenStore):
```
Comment (vidp1) [inline aries_cloudagent/askar/tests/test_profile.py:38]: test_remove_profile_not_removed_if_wallet_type_not_askar_profile

File: aries_cloudagent/askar/tests/test_profile.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        openStore = AskarOpenStore
+        context = InjectionContext() 
+        context.settings = {
+            "multitenant.wallet_type": "basic"
+        }
+        askar_profile = AskarProfile(
+            openStore,
+            context
+        )
+
```
Comment (vidp1) [inline aries_cloudagent/askar/tests/test_profile.py:48]: pls remove line

File: aries_cloudagent/askar/tests/test_profile.py [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+        remove_profile_stub = asyncio.Future()
+        remove_profile_stub.set_result(True)
+        openStore.store.remove_profile.return_value = remove_profile_stub
```
Comment (acuderman) [inline aries_cloudagent/askar/tests/test_profile.py:51]: No need to set a stub for this test case

Comment (dejsenlitro) [reply aries_cloudagent/askar/tests/test_profile.py]: Removed

File: aries_cloudagent/askar/tests/test_profile.py (hunk 3 above)
Comment (dejsenlitro) [reply aries_cloudagent/askar/tests/test_profile.py]: Removed

File: aries_cloudagent/askar/tests/test_profile.py (hunk 2 above)
Comment (dejsenlitro) [reply aries_cloudagent/askar/tests/test_profile.py]: Updated

File: aries_cloudagent/askar/tests/test_profile.py (hunk 1 above)
Comment (dejsenlitro) [reply aries_cloudagent/askar/tests/test_profile.py]: Removed

**Review by vidp1 — APPROVED** [review]

**Review by zanost — APPROVED** [review]

**Review by acuderman — APPROVED** [review]

File: aries_cloudagent/askar/tests/test_profile.py [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+            openStore,
+            context
+        )
+        remove_profile_stub = asyncio.Future()
+        remove_profile_stub.set_result(True)
+        openStore.store.remove_profile.return_value = remove_profile_stub
+
+        await askar_profile.remove()
+        
+        openStore.store.remove_profile.assert_called_once()
```
Comment (MaticDiba) [inline aries_cloudagent/askar/tests/test_profile.py:34]: Why not use `assert_called_once_with(...)`?

Comment (dejsenlitro) [reply aries_cloudagent/askar/tests/test_profile.py]: Fixed

**Pull request merged**